CHANGELOG.md

```diff
@@ -1,5 +1,127 @@
 # Version changelog
 
+## 0.23.0
+
+### Improvements and Bug Fixes
+* Introduce more specific exceptions, like `NotFound`, `AlreadyExists`, `BadRequest`, `PermissionDenied`, `InternalError`, and others ([#185](https://github.com/databricks/databricks-sdk-java/pull/185), [#257](https://github.com/databricks/databricks-sdk-java/pull/257)).
+* Lock around field accessibility changes ([#247](https://github.com/databricks/databricks-sdk-java/pull/247)).
+* Fix Changelog ([#258](https://github.com/databricks/databricks-sdk-java/pull/258)).
+* Support post with no body for APIs ([#262](https://github.com/databricks/databricks-sdk-java/pull/262)).
+
+API Changes:
+
+ * Changed `cancelRefresh()` method for `workspaceClient.lakehouseMonitors()` service with new required argument order.
+ * Changed `create()` method for `workspaceClient.lakehouseMonitors()` service with new required argument order.
+ * Changed `delete()` method for `workspaceClient.lakehouseMonitors()` service with new required argument order.
+ * Changed `get()` method for `workspaceClient.lakehouseMonitors()` service with new required argument order.
+ * Changed `getRefresh()` method for `workspaceClient.lakehouseMonitors()` service with new required argument order.
+ * Changed `listRefreshes()` method for `workspaceClient.lakehouseMonitors()` service with new required argument order.
+ * Changed `runRefresh()` method for `workspaceClient.lakehouseMonitors()` service with new required argument order.
+ * Changed `update()` method for `workspaceClient.lakehouseMonitors()` service with new required argument order.
```

Comment:
These could be merged together in single point for better readability.

Reply:
Definitely... It's just a big chore at the moment. I would like to potentially plug in `llmnotes` to condense these into a more abbreviated summary of the changes.